benchmarks/pom.xml

```diff
@@ -16,11 +16,15 @@
     <version>0.1.0-SNAPSHOT</version>
   </parent>
 
+  <properties>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
+  </properties>
+
```

Comment:
Why are you controlling Javadoc generation with a property?  Do you still want to generate Javadoc sometimes, such as during CI?